api/README.md

```diff
@@ -48,32 +50,53 @@ To authenticate to UID2 endpoints, use a bearer token in the request's authoriza
 
 ```Authorization: Bearer YourTokenBV3tua4BXNw+HVUFpxLlGy8nWN6mtgMlIk=```
 
+## Query Parameter Value Encoding
+
+When passing query parameter values in a request, ensure the query parameter value is URL-encoded. Use JavaScript's `encodeURIcomponent()` or its equivalent in your coding language.
+
 ## Email Address Normalization
 
->NOTE: The UID2 Operator Service normalizes automatically unhashed email addresses. Hashed email adresses must be normalized.
+The UID2 Operator Service normalizes unhashed email addresses automatically. You must normalize hashed email adresses before sending them in a request.
```

Comment:
Email addresses must be normalized before they are hashed. The current wording makes it sound as if hash itself needs to be normalised. The term "normalized email hash" appears in a few more places below (and also for phone numbers).

Reply:
Fixed.